Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -468,6 +464,29 @@ private bool ShouldStartBrowser()
             return true;
         }
 
+        private string GetBrowserPath()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread(nameof(GetBrowserPath));
```

Comment:
That function uses `CallerMemberNameAttribute` so the argument will be provided automatically.

Reply:
Fixed.